test/hosting/test_HostingBasic.cs

```diff
@@ -139,5 +139,37 @@ public static void TestCommandFromNative()
                 File.Delete(target);
             }
         }
+
+        [Fact]
+        public static void TestAddTypeCmdletInHostScenario()
```

Comment:
Please add a comment that describes the purpose of the test (e.g., 'Verifies reference assemblies in PowerShell hosted scenario').

Reply:
Comment added.